Find a functionally equivalent code:android.media.MediaRouter.RouteInfo.getStatusCode:COMMENT
Method Modifier: public      hidden      
Comment:/**
 * @hide
 */

@@ -1,3 +1,3 @@
 {
-    return mStatusCode;
+    return mResolvedStatusCode;
 }

